images/capi/hack/ensure-ansible.sh

```diff
@@ -39,3 +39,4 @@ ansible-galaxy collection install community.general
 ansible-galaxy collection install ansible.posix
 ansible-galaxy collection install 'ansible.windows:>=1.7.0'
 ansible-galaxy collection install community.windows
+ansible-galaxy collection install devsec.hardening
```

Comment:
Maybe we can install this via Ansible in our CaaS role in t1?

Reply:
unfortunately not that easy, i've tried to implement the ansible-galaxy module to install the hardening collection, but neither ansible-galaxy nor  ansible are found during the playbook execution. We should investigate in here, but the quick fix would be this change

Reply:
Quick fix is fine for me.